packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -988,6 +993,30 @@ export class Legend extends BaseProperties {
         // Fallback to returning closest match to the left/up.
         return closestLeftTop.datum;
     }
+    private getLegendItemHitbox(legendItemNode: { getBBox(): BBox }): BBox {
+        const bbox = legendItemNode.getBBox().clone();
+        bbox.grow(this.item.paddingX / 2, 'horizontal');
+        bbox.grow(this.item.paddingY / 2, 'vertical');
+        return bbox;
+    }
+    private getLegendItemHitboxProvider(legendItemNode: LegendMarkerLabel): BBoxProvider<BBox> {
+        // FIXME: This method is required for the `focusable` property of `createProxyElement`, but the `focusable` property should eventually be removed because it's only required to update the FocusIndicator
```

Comment:
Note: This will happen when https://github.com/ag-grid/ag-charts/pull/2418 is ready